c7n/policy.py

```diff
@@ -229,24 +229,31 @@ def run(self, *args, **kw):
             return
 
         with self.policy.ctx:
-            self.policy.log.info(
+            self.policy.log.debug(
                 "Running policy %s resource: %s region:%s c7n:%s",
                 self.policy.name, self.policy.resource_type,
                 self.policy.options.region or 'default',
                 version)
 
-            s = time.time()
-            resources = self.policy.resource_manager.resources()
-            rt = time.time() - s
+            region        = (self.policy.options.region or 'default').ljust(14, ' ')
```

Comment:
nod, understood. I think it's aesthetic preference. I can remove the ljust and rjusts.

I will say, the one biggest benefit I've had.. is when I'm parsing logs as part of testing and other situations, having that count column in the same place always is super helpful. If I'm running a test of 100 policies, it's pretty easy to scan the terminal output and see that count.

Between count, time, policy, region and resource what order do you generally prefer?

If we remove all the ljust/rjusts except for count, and put count in the first column.. it would still be easy to find which policies have count easily when scanning a large list. (This could work with no ljust or just, but having 1 ljust or just on count for say ljust(4, ' ') could be nice, or 3. Thoughts?